clickstream-event-visualiser-ui/src/main/java/clickstream/event_visualiser_ui/ui/fragment/CSEvEventListFragment.kt

```diff
@@ -0,0 +1,60 @@
+package clickstream.event_visualiser_ui.ui.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.navigation.fragment.navArgs
+import androidx.recyclerview.widget.RecyclerView
+import clickstream.event_visualiser_ui.databinding.FragmentCsEvHomeBinding
+import clickstream.event_visualiser_ui.databinding.FragmentCsEvListBinding
+import clickstream.event_visualiser_ui.ui.CSEvListItem
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+internal class CSEvEventListFragment : CSEvBaseListFragment<FragmentCsEvListBinding>() {
+
+    private val args: CSEvEventListFragmentArgs by navArgs()
+
+    override val bindingProvider: (LayoutInflater) -> FragmentCsEvListBinding
+        get() = { li ->
+            FragmentCsEvListBinding.inflate(li).also {
```

Comment:
we inflate FragmentCsEvListBinding on every single called made for `bindingProvider`?

Reply:
Just once in `onCreateView` of `CSEvBaseListFragment`